src/Styleguide/Elements/__stories__/Typography.story.tsx

```diff
@@ -22,7 +22,7 @@ stories.add("Typography", () => {
         <Flex flexDirection="column" p={2}>
           {getTypeSizes("sans").map((size, key) => {
             return (
-              <Sans size={size} key={key}>
+              <Sans size={size as any} key={key}>
```

Comment:
@alloy - I'm not sure how to coerce the items from the array into what Sans expects, since I can't cast it as a string (its expecting something concrete). Do I import `TypeSizes` from the package? And if so should we move those types top-level into `palette.d.ts` and / or export them from the root export?